client/src/index.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import SubSection from './components/SubSection.jsx';
+import MortgageSection from './components/MortgageSection.jsx';
+
+class App extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            currentProperty: {},
+            open: false
+        };
+        this.handleCollapse = this.handleCollapse.bind(this);
+    }
+
+    componentDidMount() {
+        fetch('/prices?id=1')
+            .then((res) => {
```

Comment:
how about the arrow function?
```suggestion
            .then((res) => res.json())
```